### PR TITLE
Removed drawers

### PR DESCRIPTION
**Pull request by julio8a**

### CODE REVIEW

File: dist/sass/calcite-web/utils/_overflow.scss [hunk 1]
```diff
@@ -16,9 +16,3 @@
 %overflow-auto {
   @include overflow-auto();
 }
-
-@if $include-utils == true or $include-drawers == true or $include-modal == true {
```
Comment (paulcpederson) [inline dist/sass/calcite-web/utils/_overflow.scss]: I think we'll still need these util classes

Comment (julio8a) [reply dist/sass/calcite-web/utils/_overflow.scss]: @paulcpederson, do I just remove `or $include-drawers == true`?

**Comment by paulcpederson** [comment]
Looks good but I think we should keep the utility overflow classes as they are useful even if we have no  drawers

File: dist/sass/calcite-web/utils/_overflow.scss (hunk 1 above)
Comment (paulcpederson) [reply dist/sass/calcite-web/utils/_overflow.scss]: yeah, you can just remove the check for the variable

**Comment by julio8a** [comment]
@paulcpederson, added the utility classes back. Merging.

**Pull request merged**